Lib/random.py

```diff
@@ -240,10 +240,10 @@ def __init_subclass__(cls, /, **kwargs):
     def _randbelow_with_getrandbits(self, n):
         "Return a random int in the range [0,n).  Returns 0 if n==0."
```

Comment:
```suggestion
        "Return a random int in the range [0,n).  Returns 0 if n<=1."
```

Same remark for _randbelow_without_getrandbits() docstring.

Reply:
Yeah, you are right. I've just pushed the updates to both docstrings.